lib/benchee/formatters/console/memory.ex

```diff
@@ -114,8 +115,36 @@ defmodule Benchee.Formatters.Console.Memory do
     end)
   end
 
+  @na "N/A"
+
   @spec format_scenario(Scenario.t(), unit_per_statistic, integer, boolean) :: String.t()
+  defp format_scenario(scenario, units, label_width, hide_statistics)
+
+  defp format_scenario(
+         scenario = %Scenario{memory_usage_statistics: %{sample_size: 0}},
+         _,
+         label_width,
+         _) do
+
+    warning =  "WARNING the scenario \"#{scenario.name}\" has no memory measurements!" <>
+               " This is probably a bug please report it!\n" <>
+               "https://github.com/PragTob/benchee/issues/new"
+
+    data =
+      "~*ts~*ts\n"
+      |> :io_lib.format([
+        -label_width,
+        scenario.name,
+        @average_width,
+        @na
+      ])
+      |> to_string
+
+    warning <> "\n" <> data
+  end
+
   defp format_scenario(scenario, %{memory: memory_unit}, label_width, false) do
+    # IO.inspect(scenario)
```

Comment:
Just a little thing, but this can probably go

Reply:
ugh, definitely! Thanks!